src/test/java/test/IngressTest.java

```diff
@@ -226,70 +227,167 @@ void jira_ingress_path_default(Product product) throws Exception {
+    }
+
+    @ParameterizedTest
+    @EnumSource(value = Product.class, names = "confluence")
+    void confluence_ingress_path_no_context_value_synchronyDisabled(Product product) throws Exception {
+        final var resources = helm.captureKubeResourcesFromHelmChart(product, Map.of(
+                "synchrony.enabled", "false",
+                "ingress.create", "true",
+                "ingress.host", "myhost.mydomain",
+                "ingress.path", "/ingress"));
+
+        final var ingresses = resources.getAll(Kind.Ingress);
+        Assertions.assertEquals(2, ingresses.size());
+
+        final List<String> ingressPaths = extractAllPaths(ingresses);
+
+        org.assertj.core.api.Assertions.assertThat(ingressPaths).containsExactlyInAnyOrder(
+                "/ingress",
+                "/ingress/setup",
+                "/ingress/bootstrap");
     }
 
     @ParameterizedTest
     @EnumSource(value = Product.class, names = "confluence")
     void confluence_ingress_path_default(Product product) throws Exception {
         final var resources = helm.captureKubeResourcesFromHelmChart(product, Map.of(
+                "synchrony.enabled", "true",
                 "ingress.create", "true",
                 "ingress.host", "myhost.mydomain"));
 
         final var ingresses = resources.getAll(Kind.Ingress);
         Assertions.assertEquals(2, ingresses.size());
 
-        final List<String> ingressPaths = ingresses
-                .map(ingress -> ingress.getNode("spec", "rules").required(0).path("http").path("paths").required(0).path("path").asText())
-                .collect(Collectors.toList());
-        org.assertj.core.api.Assertions.assertThat(ingressPaths).containsExactlyInAnyOrder("/", "/setup");
+        final List<String> ingressPaths = extractAllPaths(ingresses);
+
+        org.assertj.core.api.Assertions.assertThat(ingressPaths).containsExactlyInAnyOrder(
+                "/",
+                "/synchrony",
+                "/setup",
+                "/bootstrap");
     }
 
+    @ParameterizedTest
+    @EnumSource(value = Product.class, names = "confluence")
+    void confluence_ingress_path_default_synchronyDisabled(Product product) throws Exception {
+        final var resources = helm.captureKubeResourcesFromHelmChart(product, Map.of(
+                "synchrony.enabled", "false",
+                "ingress.create", "true",
+                "ingress.host", "myhost.mydomain"));
+
+        final var ingresses = resources.getAll(Kind.Ingress);
+        Assertions.assertEquals(2, ingresses.size());
 
+        final List<String> ingressPaths = extractAllPaths(ingresses);
+
+        org.assertj.core.api.Assertions.assertThat(ingressPaths).containsExactlyInAnyOrder(
+                "/",
+                "/setup",
+                "/bootstrap");
+    }
+
+    private List<String> extractAllPaths(Traversable<KubeResource> ingresses) {
+        return ingresses
+                .flatMap(ingress -> ingress.getNode("spec", "rules"))
```

Comment:
This makes tests better, because they assert all ingress paths instead of the first path for each ingress. This is why the problem was missed - because synchrony path is not the first, hence it was not covered by tests